### PR TITLE
[unit-tests] unit tests for FileDrop component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `FileDrop` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import Vuex from 'vuex'
+import vue2DropZone from 'vue2-dropzone'
+import FileDrop from '../../../../../src/components/AppBar/Upload/FileDrop.vue'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(vue2DropZone)
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js:8]: 👏

File: packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    .mockImplementation()
+  const spyOcDropzoneRemoveFiles = jest
+    .spyOn(FileDrop.methods, '$_ocDropzone_removeFiles')
+    .mockImplementation()
+
+  const store = createStore()
+  const wrapper = createWrapper({ store })
+  const dropZone = wrapper.findComponent(vue2DropZone)
+
+  it('should render component', () => {
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js:29]: please also specify when dropzone is true

Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js:29]: or wrap the whole describe to `when dropzone is enabled`

**Review by kiranparajuli589 — COMMENTED** [review]
great work

File: packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js (hunk 2 above)
Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/components/AppBar/Upload/FileDrop.spec.js:29]: also add a case to see what happens when dropzone is diabled

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

**Review by individual-it — APPROVED** [review]

**Pull request merged**